src/App.tsx

```diff
@@ -21,8 +21,10 @@ import RelevanceInspector from "./Components/RelevanceInspector";
 
 export default class App extends React.Component {
   componentDidMount() {
-    const { dispatch } = headlessEngine;
-    dispatch(SearchActions.executeSearch(AnalyticsActions.logInterfaceLoad()));
+    const { logInterfaceLoad } = loadSearchAnalyticsActions(headlessEngine);
+    const { executeSearch } = loadSearchActions(headlessEngine);
+
+    headlessEngine.dispatch(executeSearch(logInterfaceLoad()));
```

Comment:
you now have the `executeFirstSearch` method on the SearchEngine

Reply:
Publishing on `latest` only is enough,